### ISSUE TITLE
Documents tests fail to close a file handle.

### ISSUE THREAD

**Issue by thaliawww-service** [thaliawww/concrexit#590]
In GitLab by @thomwiggers on Feb 13, 2018, 12:26

### One-sentence description

Documents tests fail to close a file handle.

### Current behaviour

```
./home/thom/git/thalia/concrexit/.tox/py36-django20/lib/python3.6/site-packages/django/db/models/query.py:195: ResourceWarning: unclosed file <_io.FileIO name='/home/thom/git/thalia/concrexit/website/media/documents/file_nl_CwarPQj.pdf' mode='rb' closefd=True>
```

### Expected behaviour

No warning

### Steps to reproduce

1. tox

**Comment by thaliawww-service** [comment]
In GitLab by @thomwiggers on Feb 13, 2018, 15:08

assigned to @thomwiggers

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Feb 14, 2018, 19:31

closed via merge request !727

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Feb 14, 2018, 19:31

closed via commit 7c5b9dc7acaabccd536b894fcac6ea5713d994d6

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Feb 21, 2018, 19:14

mentioned in commit 7c5b9dc7acaabccd536b894fcac6ea5713d994d6
